.pre-commit-config.yaml

```diff
@@ -33,7 +38,13 @@ repos:
     hooks:
       - id: "yamllint"
         args: ["--format", "parsable", "--strict"]
-        exclude: ".pre-commit-config.yaml"
+        # exclude: ".pre-commit-config.yaml"
+        exclude: |
+            (?x)^(
+                .pre-commit-config.yaml|
+                custom_components/ui_lovelace_minimalist/installation/.*\.yaml|
```

Comment:
Why do you add these folders/data to be ignored by "yamllint"?

Reply:
Jinja templating.